src/cmdlib.sh

```diff
@@ -288,6 +288,36 @@ commit_overlay() {
         --skip-list <(echo /statoverride)
 }
 
+create_content_manifest(){
+        # Convert the yaml file of content_sets to json format
+    mkdir -p "${workdir}"/tmp/buildinfo
+    yaml2json /srv/redhat-coreos/content_sets.yaml "${workdir}"/tmp/buildinfo/content_manifest.json
```

Comment:
Same note here re. hardcoded path. Maybe simpler to have the caller to this function pass in the path to the YAML file.

Re. `yaml2json`, this is OK, though it doesn't look necessary. We could directly read YAML in the Python code.